stdlib/multiprocessing/pool.pyi

```diff
@@ -11,10 +11,7 @@ _T = TypeVar("_T")
 
 class ApplyResult(Generic[_T]):
     def __init__(
-        self,
-        pool: Pool,
-        callback: Optional[Callable[[_T], None]] = ...,
-        error_callback: Optional[Callable[[BaseException], None]] = ...,
+        self, pool: Pool, callback: Optional[Callable[[_T], None]], error_callback: Optional[Callable[[BaseException], None]]
```

Comment:
At least in Python 3.6, the first argument is called `cache`.

Reply:
Yeah, in 3.6 and 3.7 the cache argument is a multiprocessing.pool._PoolCache, changed in bpo-35378. We don't have typing for that, so I just moved around the stubtest allowlists instead. But you called me out on my laziness, so I'll get around to adding all the branches for 3.7 :-)

Reply:
Thanks! And feel free to merge when done.